Library/Formula/cairo.rb

```diff
@@ -34,12 +34,22 @@ def install
       --enable-gobject=yes
       --enable-svg=yes
       --enable-tee=yes
-      --enable-xlib=no
-      --enable-xlib-xrender=no
       --enable-quartz-image
     ]
 
-    args << (build.with?("x11") ? "--enable-xcb=yes" : "--enable-xcb=no")
+    args += if build.with?("x11")
```

Comment:
To keep this consistent with everything else in Homebrew I'd rather use:

```ruby
if build.with? "x11"
  args << "--enable-xcb=yes" << "--enable-xlib=yes" << "--enable-xlib-xrender=yes"
else
  args << "--enable-xcb=no" << "--enable-xlib=no" << "--enable-xlib-xrender=no"
end
```

It doesn't make the formula any wider than the URL, it uses existing style & means we can squash the arrays. Win win win, IMO.